<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpForm</class>
 <widget class="QWidget" name="HelpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Home">
       <property name="text">
        <string>Home</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/go-home.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/go-home.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Back">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/go-previous.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Forward">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/go-next.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
